### ISSUE TITLE
Scoreboard Capabilities Missing

### ISSUE THREAD

**Issue by Redigits** [drtshock/Factions#547]
Based on what I have seen in the configuration file for FactionsUUID, the following capabilities are missing:

1) The capability to show a list of individual faction members in the scoreboard
2) The capability to show the power of individual faction members in the scoreboard.
3) The capability to identify with a character like an asterisk which members in the faction are moderators and who the leader is. For example, the leader would have two asterisks (*_) and moderators would have one asterisk (_).

Are any of these capabilities possible with the plugin as it is or will custom development be required to implement these capabilities? Thank you for your help in advance.
- Redigits

**Comment by lol768** [comment]
Context: https://korobi.io/network/esper/channel/factions/logs/2015/10/18/#L426-464

**Comment by drtshock** [comment]
I don't see why we'd need to add those. What if someone had 50 people in their faction?

**Comment by Redigits** [comment]
You should be able to limit to 5-10 people, with priority given to Faction owner and then moderators so people can see all of the top ranking people in the faction followed by the rest in alphabetical order or by rank order on the server (taken from PermissionsEx). For example, you could show:

Faction Name
Total Faction Power
Your power
Faction Leader (if online)
Faction moderator 1 (if online)
Faction moderator 2  (if online)
Faction moderator ....X
Faction member (top server rank, alphabetically listed) (if online)
Faction member (second top server rank, alphabetically listed) (if online)

Many servers have it so that top rank gives special abilities. It would be good to know if those individuals are online. Similar for the owner and moderators, since they can claim land. 

CosmicPvP.me makes use of a scoreboard that shows faction members with their power. It's important for me as a faction leader or moderator to see this information because it allows me to quickly determine who is in combat in real time and who needs to stop fighting because his or her power has dropped to a dangerous level. I wouldn't be able to see that information as quickly or  as often if I had to do an /f show to see it every time.

**Comment by drtshock** [comment]
Would it only be for online faction members?

**Comment by Redigits** [comment]
That is correct. Only those who are online and in game are susceptible to losing power since they could be killed and lose power for the faction.

**Comment by Redigits** [comment]
Any timeline for this being added?

**Comment by drtshock** [comment]
Probably not very soon. I just moved across the country and am about to start a new job.
